packages/connection/legacy/class-jetpack-ixr-client.php

```diff
@@ -31,8 +32,10 @@ class Jetpack_IXR_Client extends IXR_Client {
 	 * @param int         $timeout The connection timeout, in seconds.
 	 */
 	public function __construct( $args = array(), $path = false, $port = 80, $timeout = 15 ) {
+		$connection = new Manager();
+
 		$defaults = array(
-			'url'     => Jetpack::xmlrpc_api_url(),
+			'url'     => $connection->xmlrpc_api_url(),
```

Comment:
If this the only place we're using Manager, we can do it as:
```suggestion
			'url'     => ( new Status() )->xmlrpc_api_url(),
```